tests/functional/platforms/02-host-to-platform-upgrade/flow.cylc

```diff
@@ -4,8 +4,10 @@
     UTC mode = True
 
 [scheduling]
+    initial cycle point = 2020
+    final cycle point = 2021
     [[graph]]
-        R1 = """
+        P1Y = """
```

Comment:
It's easier to compare cycle points when the tests fail if using integer cycling rather than datetime

Reply:
Agreed - I think that this should be a general principle written down somewhere.
Although I wonder if it's better to use DT cycling sometimes, since that is what many users will do.